docs/user.md

```diff
@@ -0,0 +1,193 @@
+
+[https://github.com/redhat-openshift-ecosystem/provider-certification-tool](https://github.com/redhat-openshift-ecosystem/provider-certification-tool)
+
+The provider certification tool can be used from any system with network access to the OpenShift cluster under test. 
+
+
+### Build from Source
+
+See the development guide for more information.
+*TODO - link*
+
+
+## Usage
+
+
+### Run provider certification tests
+
+```sh
+openshift-provider-cert run 
+openshift-provider-cert run -w # Keep watch open until completion
+```
+
+
+### Check status
+
+```sh
+openshift-provider-cert status 
+openshift-provider-cert status -w # Keep watch open until completion
+```
+
+
+### Retrieve the results
+
+The certification results must be retrieved from the OpenShift cluster under test using:
+
+```sh
+openshift-provider-cert retrieve
+openshift-provider-cert retrieve ./destination-dir/
+```
+
+You can see a summarized view of the results using:
+
+```sh
+openshift-provider-cert results retrieved-archive.tar.gz
+```
+
+
+### Environment Cleanup
+
+Once the certification process is complete and you are comfortable with destroying the test environment:
+
+```sh
+openshift-provider-cert destroy
+```
+
+You will need to destroy the OpenShift cluster under test separately. 
+
+
+### Certification Failures
+
+If you already know the reason for a certification failure then resolve the problem and re-run the provider certification tool again so a new certification archive is created. 
+
+If you are not sure why you have failed certification or if some of the tests fail intermittently, proceed with the troubleshooting steps below. 
+
+
+### Troubleshooting
+
+#### Information Gathering
+
+Using the _status_ command will provide a high level overview but more information is needed to troubleshoot a problem with the provider certification tool or why you aren’t able to pass all certification checks. A Must Gather from the cluster and Inspection of sonobuoy namespace is the best way to start troubleshooting:
+
+```sh
+oc adm must-gather
+oc adm inspect openshift-provider-certification
+```
+
+Use the two archives created by the commands above to begin troubleshooting. The must gather archive provides a snapshot view into the whole cluster. The inspection archive will contain information about the sonobuoy namespace only.
+
+#### Do I Need a Dedicated Test Environment
+
+When issues like this arise, you can see error events in the `openshift-provider-certification` namespace (`oc get events -n openshift-provider-certification`) or even missing plugin pods. Also, sometimes sonobuoy does not detect the issues ([SPLAT-524](https://issues.redhat.com/browse/SPLAT-524)) and the certification environment will run until the timeout, with unexpected failures.
+
+#### Running Customized Certification Plugins
+
+In some situations, you may need to modify the certification plugins that are run by the certification tool:
```

Comment:
I am not sure if we need that section in the "User documentation". I can't see scenarios where the partner needs to modify the plugin manifest to run a custom configuration. Do you have some examples?

One closer example is to set the resource requests to pods plus parallelism, maybe we can add this example. But it could result in unexpected behavior when the memory and parallelism are set too low. It's something not well tested.

In my point of view, it's more for development documentation. I also could say that it will not be supported as a certification execution.

Reply:
The other side-effect I can see doing that change is the plugin blocker workflow, as it expects specific plugins to be present in the exact order, otherwise, it will not start (it's possible to change that behavior, but need to be implemented).

Reply:
I think moving this to development document is a good idea. If someone is manipulating the plugin manifest, we would probably want to know so we can adjust the tool. I'll get this moved over!